python/lib/metrics/dmod/test/test_metrics.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python3
+import os
+import typing
+import unittest
+
+import pandas
+import numpy
+
+import dmod.metrics.metric as metrics
```

Comment:
Need to adjust this and other similar usages in the tests to something like ``from ..metrics import metric as metrics`` to avoid having to reinstall the package every time you want to test a change.